core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/components/edit-ema-palette/edit-ema-palette.component.html

```diff
@@ -0,0 +1,24 @@
+<ng-container *ngIf="vm$ | async as vm">
+    <div class="edit-ema-palette">
+        <dot-edit-ema-palette-content-type
+            *ngIf="vm.currentPaletteType === PALETTETYPE.CONTENTTYPE"
+            [contentTypes]="vm.contenttypes.items"
+            [filter]="vm.contenttypes.filter"
+            [control]="searchContenttype"
+            [isLoading]="vm.loading"
+            (showContentlets)="showContentletsFromContentType($event)"
+            (dragStart)="onDragStart($event)"
+            (dragEnd)="onDragEnd($event)" />
+
+        <dot-edit-ema-palette-contentlets
```

Comment:
If/else?

Reply:
With only one else case, i think 2 if is more easy to read than else and ng-template